scapy/contrib/cansocket.py

```diff
@@ -29,18 +29,20 @@
     log_loading.info("Configuration 'conf.contribs['CANSocket'] not found.")
 
 
+from scapy.contrib.cansocket_native import (CAN_FRAME_SIZE, CAN_INV_FILTER)  # noqa: E501 F401
```

Comment:
watch out your imports. This will import `scapy.arch.linux`, even on windows

Reply:
Sounds reasonable. Thanks for your time. I'll test it

Reply:
Fixed... Thanks a lot

Reply:
👍